python/src/DistFunc_doc.i.in

```diff
@@ -583,37 +666,421 @@ Examples
 >>> r = ot.DistFunc.rPoisson(5.0)
 >>> r = ot.DistFunc.rPoisson(5.0, 10)"
 
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::qPoisson
+"Quantile of a Poisson distribution.
+
+Parameters
+----------
+lambda : float, :math:`\lambda \geq 0`,
+    The intensity of the Poisson distribution
+p : float, in :math:`[0,1]`,
+    The probability.
+tail : bool, default=False,
+    Tail flag.
+
+    If True, the quantile associated to :math:`(1-p)` is computed.
+
+Returns
+-------
+q : float,
+    The quantile of order :math:`p` or :math:`(1-p)`.
+
+Notes
+-----
+This method is based on Mikes Giles's algorithm, described in [giles2013]_.
+
+Examples
+--------
+>>> import openturns as ot
+>>> q = ot.DistFunc.qPoisson(2.0, 0.95)"
+
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::DistFunc::dNonCentralChiSquare
 "Probability function of a NonCentralChiSquare distribution.
 
 Parameters
 ----------
-nu : float
-lambda : float
-x : float
+nu : float, :math:`\nu > 0`,
+    The :math:`\nu` parameter.
+lambda : float, :math:`\lambda \geq 0`,
+    The :math:`\lambda` parameter.
+x : float,
+    The location.
 precision : float, optional
+    The precision of the algorithm.
 maximum_iteration : int, optional
+    The max number of iterations of the algorithm.
 
 Returns
 -------
 p : float"
```

Comment:
this " should be at the end (dNonCentralChiSquare)

Reply:
Yes thank you for your help!